.gitignore

```diff
@@ -12,3 +12,4 @@ MANIFEST
 coverage.xml
 .*.sw?
 .idea/
+pip/_vendor/msgpack/*.so
```

Comment:
I will add the msgpack specific removal to `tasks/vendoring/__init__.py`. The files are interpreter and OS specific, we therefore need a glob which will not work in a patch (AFAIK).

```
pip/_vendor/msgpack/_packer.cpython-35m-darwin.so
pip/_vendor/msgpack/_unpacker.cpython-35m-darwin.so
```